pkg/sbom/collectors/mock.go

```diff
@@ -0,0 +1,70 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2024-present Datadog, Inc.
+
```

Comment:
Should we add the build tag `//go:build test` to avoid getting this being being built as part of the binary?

Reply:
Good idea ! Thank you @GustavoCaso